.gitignore

```diff
@@ -9,6 +9,9 @@
 !.config/iterm/**/*
 !.config/nvim/
 !.config/nvim/**/*
+# We DO want to ignore the plugged directories, since they contain other git repos and is used for vim plugin management
+.config/nvim/plugged/
+.config/nvim/plugged/**
```

Comment:
Oh ya I'm gonna have to migrate this dir over, or just re-install all my plugins. I guess thats fine